src/legacy/server/config/schema.js

```diff
@@ -143,7 +143,8 @@ export default () =>
           .keys({
             enabled: Joi.boolean().default(false),
             everyBytes: Joi.number()
-              .greater(1024)
+              .greater(102399)
+              .less(1073741825)
               .default(10485760),
```

Comment:
nit: could we add comments explaining what each of these numbers are (eg. `/* 100KB */`)

Reply:
👍